pkg/bmc/ilo5.go

```diff
@@ -0,0 +1,47 @@
+package bmc
+
+import (
+	"net/url"
+)
+
+type iLO5AccessDetails struct {
+	hostname string
+	path     string
+}
+
+const iLO5DefaultScheme = "https"
+
+func init() {
+	registerFactory("ilo5+virtualmedia", newiLO5VirtualMediaAccessDetails)
```

Comment:
It's not clear to me why we're creating another BMC plugin rather than just registering the existing redfish one to also handle this URL scheme?